vocabularies/Common.xml

```diff
@@ -1319,6 +1320,19 @@ Upon modification field control logic is invoked so that meta-information like h
         <Annotation Term="Core.LongDescription" String="TODO: add link to UX documentation on https://experience.sap.com/fiori-design/" />
       </Term>
 
+      <Term Name="mediaUploadURL" Type="Edm.String" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.RequiresType" String="Edm.Stream" />
+        <Annotation Term="Core.IsURL" />
+        <Annotation Term="Core.Description" String="URL for uploading new media content to a Document Management Service" />
+        <Annotation Term="Core.LongDescription">
+          <String>In contrast to the `@odata.mediaEditLink` this URL allows to upload new media content without directly changing a stream property.

```

Comment:
```suggestion
          <String>In contrast to the `@odata.mediaEditLink`, a POST request to this URL uploads new media content without directly changing a stream property.
```
How is the new content uploaded? Via a POST request to the given URL with the media stream as payload?

Reply:
Added explanation.

Reply:
But you did not explain how to upload the content.